packages/react-baseline-inputs/test/sharedExamples.ts

```diff
@@ -75,3 +76,56 @@ export const itBehavesLikeAField = (setup: FieldSetup) => {
     expect(await axe(container)).toHaveNoViolations();
   });
 };
+
+/**
+ * A set of common shared tests for a radio group.
+ */
+export const itBehavesLikeARadioGroup = (setup: RadioGroupSetup) => {
```

Comment:
This duplication probably shouldn't be necessary. Let's chat and figure out how to get the type signatures to be happy.